src/screens/DomainScreen/elements/RenderItemHeader.js

```diff
@@ -165,7 +165,7 @@ const styles = StyleSheet.create({
     borderWidth: 2
   },
   wrapperText: {
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the `backgroundColor` property might significantly alter the appearance of the component. Ensure this color change aligns with the overall design and accessibility guidelines, particularly in terms of contrast ratio for text readability.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->